plugins/network-policy-audit/tests/integration/05_run_tc_vnp_005.sh

```
@@ -0,0 +1,62 @@
+      podSelector:
+        matchLabels:
+          app: prometheus
+    ports:
+    - protocol: TCP
+      port: 8080
+EOF
+
+oc get networkpolicy -n ${NAMESPACE} > ${OUTPUT_DIR}/${TEST_ID}_policies.log 2>&1
+
+set +e  # Disable exit-on-error for plugin execution and validation
+cd ${PLUGIN_DIR}
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)